ReleaseProcessAutomation/Jira/JiraFunctionality.cs

```diff
@@ -55,6 +61,39 @@ public void CreateAndReleaseJiraVersion (SemanticVersion currentVersion, Semanti
     ReleaseVersion(currentVersionID, nextVersionID, squashUnreleased);
   }
 
+  public bool MoveClosedIssuesToNewVersion (SemanticVersion currentVersion, SemanticVersion nextVersion)
+  {
+    var currentJiraVersion = _jiraVersionCreator.FindVersionWithVersionNumber(currentVersion.ToString());
+    var nextJiraVersion = _jiraVersionCreator.FindVersionWithVersionNumber(nextVersion.ToString());
+    if(currentJiraVersion == null || nextJiraVersion == null)
+    {
+      _console.WriteLine($"Could not find current jira version '{currentVersion}' or next jira version '{nextVersion}'");
+      return false;
+    }
+    
+    var closedIssues = _jiraIssueService.FindAllClosedIssues(currentJiraVersion.id);
+    _jiraIssueService.MoveIssuesToVersion(closedIssues, currentJiraVersion.id, nextJiraVersion.id);
+    return true;
```

Comment:
As it is used in `ReleaseAlphaBetaStep` this method should move all the issues listed in the `GetClosedVersionsSummaryWithExactFixVersion` method but the logic is a lot simpler. I think we are missing something here. Maybe it is better if we pass in the jira issues we want to have moved and the new version.